Makefile

```diff
@@ -202,7 +202,7 @@ kindleupdate: all
 	# note that the targz file extension is intended to keep ISP from caching
 	# the file, see koreader#1644.
 	cd $(INSTALL_DIR) && \
-		tar -I"gzip --rsyncable" -cah --no-recursion -f ../$(KINDLE_PACKAGE_OTA) \
+		tar --hard-dereference -I"gzip --rsyncable" -cah --no-recursion -f ../$(KINDLE_PACKAGE_OTA) \
```

Comment:
This is technically only for Kindle?

(I'm fine keeping it the same everywhere for simplicity; just asking.)

Reply:
Yeah, although I figured this couldn't hurt, since most of those will be deployed on vfat anyway, and all of them were already using `--dereference` (`h`).